css/interactive-guides/main.scss

```diff
@@ -176,13 +176,7 @@ note {
   visibility: hidden;
 }
 
-#copy_to_clipboard {
-  position: absolute;
-    display: block;
-    background-color: #eeeff3;
-    background-image: url(/img/guide_copy_button.svg);
-    background-repeat: no-repeat;
-    background-position: center center;
-    height: 50px;
-    width: 38px;
-}
+#copy_to_clipboard {	
+  background-color: #FFFFFF;	
+  background-image: url(/img/guides_copy_button.svg);	
```

Comment:
Is it needed to repeat the background-image here?  Won't it just pick up the one from \#copy_to_clipboard in guide.css?  I'd remove it from here if not really needed so a change in the image in the future only requires an update to only one css file.